aws/ami/files/scylla_install_ami

```diff
@@ -39,6 +40,7 @@ if __name__ == '__main__':
     if os.getuid() > 0:
         print('Requires root permission.')
         sys.exit(1)
+    distro = platform.linux_distribution(full_distribution_name=False)[0].lower()
```

Comment:
`platform.linux_distribution` is deprecated and removed in Python 3.7
https://docs.python.org/3.5/library/platform.html#platform.linux_distribution

Reply:
The script is unfortunately written in python2, because we use it for running `yum install scylla` in CentOS7.
After we installed scylla-python3 we can use newest python3, but before installing it's impossible.

Maybe writing in python was mistake at all, better to re-write in bash, since we couldn't find any python interpreter in Ubuntu 20.04LTS AMI.

Reply:
Ubuntu 20.04 has Python 3.8.5 installed...
![image](https://user-images.githubusercontent.com/5269241/107943422-bf99ac00-6f9d-11eb-8b8c-0636d86ed509.png)

Reply:
Ahh, I was wrong, cloud-init requires python3 so Ubuntu AMI should have python3 interpreter. So the script should written in Python 2-3 compatible code.

Reply:
https://packages.ubuntu.com/focal/cloud-init

Reply:
rewrited scylla_install_ami for python2-3 compatible code, run it on python3 on Ubuntu AMI.
Dropped platform.linux_distribution() since it doesn't work on python3.